cookbooks/aws-parallelcluster-environment/recipes/config/export_home.rb

```diff
@@ -14,6 +14,18 @@
 
 return if on_docker?
 
+return unless node['cluster']['internal_shared_storage_type'] == 'ebs'
```

Comment:
What if the /home is the mountdir of an EBS shared storage and the internal shared storage is EBS?
We would end up skipping to export the /home in that case, that is not desirable.

Reply:
Yes, I remembered this a few minutes ago. We need a validator to disable EBS as an option in shared storage for /home. This effort is being tracked internally as a followup before the release